### PR TITLE
Pr/update healthkit answerformat

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
https://github.com/ResearchKit/ResearchKit/pull/865

### CODE REVIEW

File: ResearchKit/Common/ORKAnswerFormat.h [hunk 1]
```diff
@@ -168,6 +168,16 @@ ORK_CLASS_AVAILABLE
  */
 - (void)validateParameters;
 
+/**
+ Some answer formats are constructed of other answer formats. This method allows
+ a subclass to return a different answer format for use in defining the UI/UX for
+ the answer format type. For example, a Boolean answer format is presented in the 
+ same way as a single-choice answer format with the choices Yes and No mapping to 
+ `@(YES)` and `@(NO)`, respectively, so it's `impliedAnswerFormat` is an
```
Comment (Erin-Mounts) [inline ResearchKit/Common/ORKAnswerFormat.h]: its not it's

File: ResearchKit/Common/ORKHealthAnswerFormat.m [hunk 2]
```diff
@@ -172,6 +197,21 @@ - (ORKAnswerFormat *)impliedAnswerFormat {
...
+                                 [ORKTextChoice choiceWithText:ORKLocalizedString(@"FITZPATRICK_SKIN_TYPE_I", nil) value:@(HKFitzpatrickSkinTypeI)],
+                                 [ORKTextChoice choiceWithText:ORKLocalizedString(@"FITZPATRICK_SKIN_TYPE_II", nil) value:@(HKFitzpatrickSkinTypeII)],
+                                 [ORKTextChoice choiceWithText:ORKLocalizedString(@"FITZPATRICK_SKIN_TYPE_III", nil) value:@(HKFitzpatrickSkinTypeIII)],
+                                 [ORKTextChoice choiceWithText:ORKLocalizedString(@"FITZPATRICK_SKIN_TYPE_IV", nil) value:@(HKFitzpatrickSkinTypeIV)],
+                                 [ORKTextChoice choiceWithText:ORKLocalizedString(@"FITZPATRICK_SKIN_TYPE_V", nil) value:@(HKFitzpatrickSkinTypeV)],
+                                 [ORKTextChoice choiceWithText:ORKLocalizedString(@"FITZPATRICK_SKIN_TYPE_VI", nil) value:@(HKFitzpatrickSkinTypeVI)],
+                                 ];
+            ORKValuePickerAnswerFormat *format = [ORKAnswerFormat valuePickerAnswerFormatWithTextChoices:options];
+            _impliedAnswerFormat = format;
+        }
```
Comment (Erin-Mounts) [inline ResearchKit/Common/ORKHealthAnswerFormat.m]: consistent brace formatting please

File: ResearchKit/Common/ORKHealthAnswerFormat.m [hunk 3]
```diff
@@ -284,9 +336,9 @@ - (ORKAnswerFormat *)impliedAnswerFormat {
         ORKNumericAnswerFormat *format = nil;
             HKUnit *unit = [self healthKitUserUnit];
         if (_numericAnswerStyle == ORKNumericAnswerStyleDecimal) {
-            format = [ORKNumericAnswerFormat decimalAnswerFormatWithUnit:[unit unitString]];
+            format = [ORKNumericAnswerFormat decimalAnswerFormatWithUnit:[unit localizedUnitString]];
```
Comment (Erin-Mounts) [inline ResearchKit/Common/ORKHealthAnswerFormat.m:337]: Are these strings for display to the participant, or eventually uploaded to Bridge as part of the answer? If the former, cool; if the latter, researchers in the past have requested they be in one standard localization so data from all users in Synapse is consistent regardless of locale. I'm guessing from the comment below that you're already aware of this, but just being extra sure.

**Review by Erin-Mounts — APPROVED** [review]
A typo, a formatting request, and one question, otherwise looks good, merge when ready

File: ResearchKit/Common/ORKHealthAnswerFormat.m (hunk 3 above)
Comment (syoung-smallwisdom) [reply ResearchKit/Common/ORKHealthAnswerFormat.m:337]: Thanks for asking! (I couldn't remember anymore so had to check) As it turns out, the way this works is that this is in the `impliedAnswerFormat` which is the answer format used to display the question to the user. The results are saved using `-resultWithIdentifier:answer:` which uses `unitString` instead.

**Pull request merged**